seed/seedwriter/writer.go

```diff
@@ -864,6 +878,12 @@ func (w *Writer) snapDecl(sn *SeedSnap) (*asserts.SnapDeclaration, error) {
 	return nil, fmt.Errorf("internal error: snap %q has no snap-declaration set", sn.SnapName())
 }
 
+// Warnings returns the warning messages produced so far. No warnings
+// should be generated after Downloaded signaled complete.
+func (w *Writer) Warnings() []string {
```

Comment:
I reckon this is unrelated to our warnings API; should we name it differently to avoid any confusion?

Reply:
it's unrelated, but at some point we have to trust people to be able to do context, it might be better than to invent more weird names for things